erpnext/public/js/controllers/buying.js

```diff
@@ -422,6 +359,62 @@ erpnext.buying.BuyingController = erpnext.TransactionController.extend({
 
 cur_frm.add_fetch('project', 'cost_center', 'cost_center');
 
+erpnext.buying.link_to_mrs = function(frm) {
+	frappe.call({
+		method: "erpnext.buying.utils.get_linked_material_requests",
+		args:{
+			items: frm.doc.items.map((item) => {return item.item_code;})
+		},
+		callback: function(r) {
+			if(!r.message || r.message.length == 0) {
+				frappe.throw({message: __("No pending Material Requests found to link for the given items."), title: __("Note")});
+			}
+			else {
```

Comment:
We can remove the `else` block if we're returning from above `if` block. Not needed.